build/build.sh

```diff
@@ -91,12 +91,11 @@ build(){
                 output_name+='.exe'
             fi
             echo "Building for: ${GOOS} ${GOARCH}"
-            gox -cgo -os="$GOOS" -arch="$GOARCH" -ldflags \
-                "-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
+            go build \
+                -ldflags="-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
                 -X main.CtlName='${CTLNAME}' \
-                -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
-                -output="$output_name" \
-                ./cmd/"$BUILDPATH"
+                -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION} " \
```

Comment:
Here also, the additional space character can be removed.
```suggestion
                -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
```